modules/project/vars.tf

```diff
@@ -35,6 +35,19 @@ variable default_service_account {
   default     = "deprivilege"
 }
 
+## Terraform state bucket
+variable "tribe" {
```

Comment:
Follow the same pattern as for existing vars and remove the quotes around the name.